### PR TITLE
Update proxy server examples to include a diff ip address

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Update proxy server examples to include a different ip address

### CODE REVIEW

File: network/proxy.md [hunk 1]
```diff
@@ -11,32 +11,30 @@ configure it in different ways:
   [configure the Docker client](#configure-the-docker-client) to pass
   proxy information to containers automatically.
 
-- In Docker 17.06 and lower, you must
-  [set appropriate environment variables](#use-environment-variables)
+- In Docker 17.06 and earlier versions, you must set the appropriate
```
Comment (usha-mandya) [inline network/proxy.md:14]: @StefanScherer Not sure if we should still make this distinction as 17.06 is quite old.

File: network/proxy.md [hunk 2]
```diff
@@ -11,32 +11,30 @@ configure it in different ways:
...
     ```json
     {
      "proxies":
      {
        "default":
        {
-         "httpProxy": "http://127.0.0.1:3001",
-         "httpsProxy": "http://127.0.0.1:3001",
+         "httpProxy": "http://192.168.1.12:3128",
+         "httpsProxy": "https://192.168.1.12:3128",
```
Comment (StefanScherer) [inline network/proxy.md:37]: Keep http here

**Review by StefanScherer — DISMISSED** [review]
LGTM

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**